### ISSUE TITLE
HA Testing: Create manual test cases

### ISSUE THREAD

**Issue by MarkAckert** [zowe/zowe-install-packaging#1476]
* Design test cases to validate HA solutions and detect availability interruptions.
  - Re-use some of the performance test suite / test cases
  - Distinguish test results of service completely shutdown and downgraded. If it's downgraded, we need to evaluate if this is acceptable, and if this downgrade will cause any snowfall issue.
* Chaos testing - kill a server, inject network failures
  - Kill a LPAR, what happens?

Further details to be filled out once we know what our HA solution is :)

**Comment by jackjia-ibm** [comment]
For 21PI1, the goal is to define manual test scenarios and verify the release manually.

**Comment by jackjia-ibm** [comment]
Manual test plan defined here https://github.com/zowe/zowe-install-packaging/wiki/Test-Zowe-in-Sysplex

**Comment by jackjia-ibm** [comment]
Changed this task to manual testing. Will create new task for automation in the future once Sysplex on Marist is ready.
